chef/cookbooks/glance/templates/default/glance-api.conf.erb

```diff
@@ -387,6 +387,7 @@ identity_uri = <%= @keystone_settings['protocol'] %>://<%= @keystone_settings['i
 admin_tenant_name = <%= @keystone_settings['service_tenant'] %>
```

Comment:
Btw, why don't we use `<%= @keystone_settings['admin_auth_url'] %>` for `identity_uri` at https://github.com/crowbar/barclamp-glance/pull/252/files#diff-86b6a6b811558c57ef953fa962833ccaR386 ?